src/index.js

```diff
@@ -166,19 +172,32 @@ exports.createUploadLink = ({
       })
 
       options.body = form
-    } else options.body = payload
+    } else {
+      // If requested, set method to GET if there are no mutations.
+      const definitionIsMutation = d =>
```

Comment:
Let's move this function directly into the `.some()`; holding it in a variable is not necessary as it's only used in one place.

Also, let's rename `d` to something more descriptive, like `definition`.